Provide full resources  in archived items finished webhooks

tw4l opened the pull request:
Fixes #1306 

- Include full `resources` with expireAt (as string) in crawlFinished and uploadFinished webhook notifications rather than using the `downloadUrls` field (this is retained for collections).
- Set default presigned duration to just short of 1 week and enforce maximum supported by S3
- Update tests

Tested locally and on dev

tw4l committed:
Provide full resources  in archived items finished webhooks

- Include full resources with expireAt (as string) in crawlFinished
and uploadFinished webhook notifications rather than using the
downloadUrls field (this is retained for collections).
- Set default presigned duration to just short of 1 week and
enforce maximum supported by S3
- Update tests

ikreymer commented on backend/btrixcloud/basecrawls.py:
```diff
@@ -62,8 +67,12 @@ def __init__(
-        self.presign_duration_seconds = (
-            int(os.environ.get("PRESIGN_DURATION_MINUTES", 60)) * 60
+        presign_duration_seconds = (
+            int(os.environ.get("PRESIGN_DURATION_MINUTES", PRESIGN_MINUTES_DEFAULT))
```
We also have:
```
  PRESIGN_DURATION_MINUTES: "{{ .Values.storage_presign_duration_minutes | default 60 }}"
```
in configmap.yaml so that will take precedence. probably should keep this default in one place only to avoid confusion, can make the env var blank by default

ikreymer replied:
oh, I see its been updated to latest value, but should it be blank there perhaps?

ikreymer commented on backend/btrixcloud/basecrawls.py:
```diff
@@ -62,8 +67,12 @@ def __init__(
+            int(os.environ.get("PRESIGN_DURATION_MINUTES", PRESIGN_MINUTES_DEFAULT))
+            * 60
+        )
+        self.presign_duration_seconds = min(
```
Maybe we should just apply the maximum to minutes, one less value to keep track of, so it'll be a max of 10079 * 60

ikreymer committed:
clean up presign defaults:
- set default minutes in code, if omitted in chart
- add commented out value to chart
- use minutes as maximum

ikreymer committed:
fix default
